cpp/Platform.Converters/CachingConverterDecorator.h

```diff
@@ -1,15 +1,28 @@
-﻿namespace Platform::Converters
+﻿#pragma once
+#include <unordered_map>
+#include <memory>
+
+#include "IConverter[TSource, TTarget].h"
```

Comment:
`IConverter[TSource, TTarget].h` does not exist

1. 

 in C++ version of code.